### DISCUSSION TITLE
My images will not render (but alt text appears).

### DISCUSSION THREAD

**Discussion by SethBurns** [vercel/next.js#56752] [Help]
### Summary

I have my images in the public folder. I'm attempting to use the Image component from next/image.
I'm importing my image directly from the public folder and then using the alias I create as the src={importedImage}.

The error I keep getting is: 'The requested resource isn't a valid image for /_next/static/media/yssdivers.04b08fe2.jpg received text/html; charset=utf-8'

When I navigate to that path in my project, I see that file, and it is in fact a .jpg that shows up in my VSCode preview no problem. So it IS a valid image... if I change the 'Image /' component to just an 'img /' then it all works just fine and as expected. I cannot for the life of me figure this out. I've read so many Stack Overflow posts and Github threads that I'm starting to go crazy.

I'm happy to share more code if needed. I've shared the code from the page that seems to be the culprit of my issues. Also attaching a picture of my file explorer.

![Screen Shot 2023-10-12 at 8 30 02 AM](https://github.com/vercel/next.js/assets/123792434/5e89713d-06e0-483f-8ebc-0a397b9c3f24)

### Additional information

```js
'use client';
import { Form } from './_components/Form';
import { Nav } from './_components/Nav';
import Image from 'next/image';
import yssDivers from '../../public/yssdivers.jpg'


export default function Home() {
  return (
    <main className="flex min-h-screen flex-col items-center p-2">
      {/* eslint-disable-next-line @next/next/no-img-element */}
      <Image
        alt="YSS Divers Entering Hood Canal"
        src={yssDivers}
        style={{
          objectFit: 'cover',
          zIndex: -1,
          position: 'fixed',
        }}
      />
        <Nav />
        <Form />
    </main>
  );
}
```

**Comment by icyJoseph** [comment]
I can't see in your three structure, but do you maybe have middleware or something else that would be catching the request and perhaps returns HTML of some sort? What do you see if you navigate to `yourdomain.com/_next/static/media/yssdivers.04b08fe2.jpg`?

**Comment by okoraretega** [comment]
Have you been able to fix this issue? cause I'm getting the same error on my project

**Comment by AndyJPhillips-lively** [comment]
For my scenario I had an nginx server running in the background that was causing some issues. Killing the nginx server solved the issue.

**Comment by adriansasu** [comment]
Same here. Getting the same error and have not yet found a workaround to fix
